### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by Swyatik19**

### CODE REVIEW

File: src/main/java/mate/jdbc/Main.java [hunk 1]
```diff
@@ -1,7 +1,32 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerDao manufacturerService = (ManufacturerDao) injector
```
Comment (karma-o) [inline src/main/java/mate/jdbc/Main.java:9]:
```suggestion
    private static final ManufacturerDao manufacturerDao = (ManufacturerDao) injector
```

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturer(name, country) values(?, ?);";
+
+        try (Connection connection = ConnectivityUtil.getConnection();
+                PreparedStatement createManufacturerStatement = connection
+                        .prepareStatement(insertManufacturerRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1, manufacturer.getName());
+            createManufacturerStatement.setString(2, manufacturer.getCountry());
```
Comment (karma-o) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:44]: magic numbers, check other places as well

**Review by karma-o — CHANGES_REQUESTED** [review]
Add the missing `init_db.sql` file!

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getRequest = "SELECT * FROM manufacturer WHERE id = ? AND is_deleted = false";
+
+        try (Connection connection = ConnectivityUtil.getConnection();
+
```
Comment (Rommelua) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:66]: empty line

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturer SET name = ?, country = ?"
+                + " where id = ? AND is_deleted = false";
+        try (Connection connection = ConnectivityUtil.getConnection();
+                PreparedStatement updateStatement = connection.prepareStatement(updateRequest)) {
+            updateStatement.setString(NAME_MANUFACTURER, manufacturer.getName());
+            updateStatement.setString(COUNTRY_MANUFACTURER, manufacturer.getCountry());
+            updateStatement.setLong(ID_MANUFACTURER, manufacturer.getId());
+            updateStatement.executeUpdate();
+
```
Comment (Rommelua) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:91]: empty line

File: src/main/java/mate/jdbc/util/ConnectivityUtil.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public static Connection getConnection() {
+
+        try {
+            Properties properties = new Properties();
+            properties.put("user", "root");
+            properties.put("password", "********");
+            return DriverManager.getConnection("jdbc:mysql://localhost:3306/taxi_service_db",
+                    properties);
```
Comment (Rommelua) [inline src/main/java/mate/jdbc/util/ConnectivityUtil.java:24]: user, password and url should be constants

**Review by Rommelua — APPROVED** [review]

**Pull request closed without merging**